iree/vm/test/call_ops.mlir

```diff
@@ -0,0 +1,46 @@
+vm.module @call_ops {
+
+  vm.export @test_call_void
+  vm.func @test_call_void() {
+    vm.call @_v_v() : () -> ()
+    // %0 = vm.call @_v_i32() : () -> i32
+    vm.return
+  }
+
+  vm.export @test_call_i
+  vm.func @test_call_i() {
+    %c1 = vm.const.i32 1 : i32
+    %0 = vm.call @_v_i() : () -> (i32)
+    vm.check.eq %0, %c1, "_v_i32()=1" : i32
+    vm.return
+  }
+
+  vm.export @test_call_ii
+  vm.func @test_call_ii() attributes {emitc.disable} {
+    %c1 = vm.const.i32 1 : i32
+    %c2 = vm.const.i32 2 : i32
+    %0:2 = vm.call @_v_ii() : () -> (i32, i32)
+    vm.check.eq %0#0, %c1, "_v_ii()#0=1" : i32
+    vm.check.eq %0#1, %c2, "_v_ii()#1=2" : i32
+    vm.return
+  }
+
+  vm.func  @_v_v() attributes {noinline} {
```

Comment:
remove extra space after vm.func (here and below)
(I wish there was a clang-format for mlir!)